driver/web/docs/schemas/application/SurveyData.yaml

```diff
@@ -0,0 +1,85 @@
+type: object
+properties:
+  id:
+    type: string
+  key:
+    type: string
+  section:
+    type: string
+  allow_skip:
+    type: string
+  text:
+    type: array
+    items:
+      type: string
+  more_info:
+    type: string  
+  default_follow_up_key:
+    type: string
+  default_response_rule:
+    type: string      
+  follow_up_rule:
+    type: string
+  score_rule:
+    type: boolean
+  response:
+    type: object
+  type:
+    type: string
+  ok_answer:
+    type: object
+    nullable: true
+  ok_answers:
+    type: array
+    items: {}
+    nullable: true
+  options:
+    type: array
+    items: {}
+  nullable: true
+  action:
+    $ref: "./ActionData.yaml"
+    nullable: true
+  yes_no:
+    type: boolean
+    nullable: true
+  allow_multiple:
+    type: boolean
+    nullable: true
+  check_all:
+    type: boolean
+    nullable: true
+  ask_time:
+    type: boolean
+  minimum:
```

Comment:
Watch out for extra indentation (all fields below `minimum` have extra indentation)